lib/matplotlib/text.py

```diff
@@ -874,7 +874,7 @@ def get_window_extent(self, renderer=None, dpi=None):
         """
         #return _unit_box
         if not self.get_visible():
-            return Bbox.unit()
```

Comment:
Old behavior was that text took up exactly one pixel (at its xy coords) whether or not the text was visible. The text should take up *no* space if it's not drawn.